src/graphql/admin/mutations.ts

```diff
@@ -1,34 +1,70 @@
+import { getAccountsConfig } from "@config"
+
 import { GT } from "@graphql/index"
 
 import UserRequestAuthCodeMutation from "@graphql/root/mutation/user-request-auth-code"
 import UserLoginMutation from "@graphql/root/mutation/user-login"
 import CaptchaRequestAuthCodeMutation from "@graphql/root/mutation/captcha-request-auth-code"
 import CaptchaCreateChallengeMutation from "@graphql/root/mutation/captcha-create-challenge"
-
 import AccountUpdateLevelMutation from "@graphql/admin/root/mutation/account-update-level"
 import AccountUpdateStatusMutation from "@graphql/admin/root/mutation/account-update-status"
+import AccountsAddUsdWalletMutation from "@graphql/admin/root/mutation/account-add-usd-wallet"
+import AccountCustomFieldsUpdateMutation from "@graphql/admin/root/mutation/account-custom-fields-update"
 import BusinessUpdateMapInfoMutation from "@graphql/admin/root/mutation/business-update-map-info"
 import ColdStorageRebalanceToHotWalletMutation from "@graphql/admin/root/mutation/cold-storage-rebalance-to-hot-wallet"
 
-import AccountsAddUsdWalletMutation from "./root/mutation/account-add-usd-wallet"
+import {
+  addAttributesToCurrentSpanAndPropagate,
+  SemanticAttributes,
+  ACCOUNT_USERNAME,
+} from "@services/tracing"
 
-const MutationType = GT.Object({
-  name: "Mutation",
-  fields: () => ({
-    userRequestAuthCode: UserRequestAuthCodeMutation,
-    userLogin: UserLoginMutation,
+const fields = {
+  userRequestAuthCode: UserRequestAuthCodeMutation,
+  userLogin: UserLoginMutation,
+
+  captchaCreateChallenge: CaptchaCreateChallengeMutation,
+  captchaRequestAuthCode: CaptchaRequestAuthCodeMutation,
 
-    captchaCreateChallenge: CaptchaCreateChallengeMutation,
-    captchaRequestAuthCode: CaptchaRequestAuthCodeMutation,
+  accountUpdateLevel: AccountUpdateLevelMutation,
+  accountUpdateStatus: AccountUpdateStatusMutation,
+  accountsAddUsdWallet: AccountsAddUsdWalletMutation,
 
-    accountUpdateLevel: AccountUpdateLevelMutation,
-    accountUpdateStatus: AccountUpdateStatusMutation,
-    accountsAddUsdWallet: AccountsAddUsdWalletMutation,
+  businessUpdateMapInfo: BusinessUpdateMapInfoMutation,
 
-    businessUpdateMapInfo: BusinessUpdateMapInfoMutation,
+  coldStorageRebalanceToHotWallet: ColdStorageRebalanceToHotWalletMutation,
+}
 
-    coldStorageRebalanceToHotWallet: ColdStorageRebalanceToHotWalletMutation,
-  }),
+const { customFields } = getAccountsConfig()
+if (customFields && customFields.length > 0) {
+  Object.assign(fields, { accountCustomFieldsUpdate: AccountCustomFieldsUpdateMutation })
```

Comment:
When I was thinking about it, I had a different idea in mind in regards to the API. I guess this is related to the code review discussion we had Tuesday, that we may want to have specific use case for mutation.

instead of having a generic `AccountCustomFieldsUpdateMutation`,
we could create one API endpoint per customField

for instance, we could have:
```
AccountSetFirstname
AccountSetLastname
```
etc.

the result would be that the implementation of the custom field doesn't explicitly appear in the API, which I think is better. 

but this is just my thoughts. we should probably gain some consensus on this one.

Reply:
Changes according to code review https://github.com/GaloyMoney/galoy/pull/1465/commits/7ee35631e7b7da09da9382ced0a46dd02960d8a6